static/src/javascripts/projects/commercial/modules/dfp/define-slot.ts

```diff
@@ -29,11 +29,10 @@ const isBreakpoint = (
 /**
  * Builds a googletag size mapping based on the breakpoints and ad sizes from
  * the size mapping in commercial-core and the viewport sizes from source-foundations.
- *
  */
 const buildGoogletagSizeMapping = (
```

Comment:
This function can return null. The info for `mapping.build()` includes the following:

_@returns — The result built by this builder. Can be null if invalid size mappings were supplied._

Reply:
If this is the case it might be worth us raising a PR against the [type definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/googletag/index.d.ts#L1100) that we use!